dwave/system/composites/virtual_graph.py

```diff
@@ -22,73 +27,206 @@
-        ...     @property
-        ...      def nodelist(self):
-        ...         return [0, 1, 2]
-        ...
-        ...     @property
-        ...     def edgelist(self):
-        ...         return [(0, 1), (1, 2)]
-        >>> test_obj = StructuredObject()
-        >>> for u, v in test_obj.edgelist:
-        ...     assert u in test_obj.adjacency[v]
-        ...     assert v in test_obj.adjacency[u]
+       This example uses :class:`.VirtualGraphComposite` to instantiate a composed sampler
+       that uses a D-Wave solver selected by the user's default D-Wave Cloud Client configuration_ file.
+       Because qubits 0, 5, and coupled qubits {0, 4} are all coupled on the selected D-Wave solver, edges
+       between three nodes, x, y, and z, as specified by the embedding, are available to problems using this
+       composed sampler. However, qubit 8 is in an adjacent unit cell on the D-Wave solver and not directly
+       connected to the other four qubits, so node `a` does not share an edge with any other nodes.
+
+       >>> from dwave.system.samplers import DWaveSampler
+       >>> from dwave.system.composites import VirtualGraphComposite
+       >>> embedding = {'x': {1}, 'y': {5}, 'z': {0, 4}, 'a': {8}}
+       >>> sampler = VirtualGraphComposite(DWaveSampler(), embedding)  # doctest: +SKIP
+       >>> sampler.adjacency  # doctest: +SKIP
+       {'a': set(), 'x': {'y', 'z'}, 'y': {'x', 'z'}, 'z': {'x', 'y'}}
+
+    .. _configuration: http://dwave-cloud-client.readthedocs.io/en/latest/#module-dwave.cloud.config
 
     """
 
     children = None
-    """list: A list containig the wrapped sampler."""
+    """list: List containing the wrapped sampler."""
 
     parameters = None
-    """The same parameters as are accepted by the child sampler with an additional parameter
-    'apply_flux_bias_offsets'.
+    """dict[str, list]: Parameters in the form of a dict.
+
+    For an instantiated composed sampler, keys are the keyword parameters accepted by the child
+    sampler with an additional parameter, 'apply_flux_bias_offsets'.
+
+    Examples:
+       This example uses :class:`.VirtualGraphComposite` to instantiate a composed sampler
+       that uses a D-Wave solver selected by the user's default D-Wave Cloud Client configuration_ file
+       and views the composed sampler's parameters.
+
+       >>> from dwave.system.samplers import DWaveSampler
+       >>> from dwave.system.composites import VirtualGraphComposite
+       >>> embedding = {'x': {1}, 'y': {5}, 'z': {0, 4}}
+       >>> sampler = VirtualGraphComposite(DWaveSampler(), embedding)  # doctest: +SKIP
+       >>> sampler.parameters  # doctest: +SKIP
+       {u'anneal_offsets': ['parameters'],
+        u'anneal_schedule': ['parameters'],
+        u'annealing_time': ['parameters'],
+        u'answer_mode': ['parameters'],
+        'apply_flux_bias_offsets': [],
+        u'auto_scale': ['parameters'],
+       >>>  # Snipped above response for brevity
```

Comment:
In this case the u implies unicode string which is default for python3. So for the examples we should show the python3 output (no 'u's)